coderd/database/queries/appusage.sql

```diff
@@ -0,0 +1,29 @@
+-- name: InsertAppUsage :one
+INSERT INTO
+	app_usage (
+		user_id,
+		template_id,
+		app_id,
+		created_at
+	)
+VALUES
+	($1, $2, $3, $4) RETURNING *;
+
+-- name: GetAppUsageByDate :one
+SELECT
+	*
+FROM
+	app_usage
+WHERE
+	user_id = $1 AND template_id = $2 AND app_id = $3 AND created_at = $4;
```

Comment:
Could this be an insert with an "ON CONFLICT" clause instead? Avoids having to select and then insert perhaps. I'm unsure if we need more indices for this to work though.

Reply:
I will take a look on the "ON CONFLICT"